### PR TITLE
Emp 255

### PR DESCRIPTION
**Pull request by solomonmanik**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/LASB-XXX)

Describe what you did and why.

## Checklist

Before you ask people to review this PR:

- [ ] Tests should be passing: `./gradlew test`
- [ ] Github should not be reporting conflicts; you should have recently run `git rebase main`.
- [ ] Avoid mixing whitespace changes with code changes in the same commit. These make diffs harder to read and conflicts more likely.
- [ ] You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- [ ] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: data-api/schemas/crm4.yml [hunk 1]
```diff
@@ -103,7 +101,7 @@ Crm4CaseDetailsDTO:
       $ref: '#/Crm4ClientDTO'
     ProceedingDetails:
       $ref: '#/Crm4ProceedingDetails'
```
Comment (tanovellino) [inline data-api/schemas/crm4.yml:103]: This should be named `Crm4ProceedingDetailsDTO` as the rest

File: equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm4Mapper.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    @Mapping(target="caseDetails.firm.firmContactName", source="contact_name")
+    @Mapping(target="caseDetails.firm.firmSolicitorName", source="solicitorname")
+
+    @Mapping(target="caseDetails.solicitorDetails.solicitorName", source="solicitorname")
+    @Mapping(target="caseDetails.solicitorDetails.solicitorReference", source="solicitorid")
+
+    @Mapping(target="caseDetails.clientDetails.ufn", source="client_ufn")
+    @Mapping(target="caseDetails.clientDetails.maatNumber", source="maat_number")
+    @Mapping(target="caseDetails.clientDetails.firstName", source="client_forename")
+    @Mapping(target="caseDetails.clientDetails.middleName", expression = "java(null)")
```
Comment (tanovellino) [inline equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm4Mapper.java:29]: Just as a remainder, for both of us. On those fields we are currently adding `null`, we need to get confirmation from BAs if they are still required, should be avoided or if they should be added to the Model even those the field does not exists in our test XML

**Review by tanovellino — APPROVED** [review]
LGTM. The renaming part can be easily pushed with the next commit to be done as part of the same ticket

File: equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm4Mapper.java (hunk 2 above)
Comment (solomonmanik) [reply equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm4Mapper.java:29]: I'll remove middlename it is not returned as part of existing data.

File: data-api/schemas/crm4.yml (hunk 1 above)
Comment (solomonmanik) [reply data-api/schemas/crm4.yml]: Done.Thanks

**Pull request merged**